master/static/srv/get_experiment.sql

```diff
@@ -1,5 +1,9 @@
 SELECT
     e.id AS id,
+    /* TODO We could either do this and rely that we go through the apis that use this query
+     (eg update cli) or migrate older experiments in database (one way). Look at model defaults */
+    COALESCE(NULLIF(e.config->>'name', ''), NULLIF(e.config->>'description', ''), 'Experiment ' ||  e.id) AS name,
```

Comment:
This logic could happen here, but I think it's probably better to write a shim for the exp conf and trust that all old expconfs are shimmed on startup, rather than scattering shimming logic all over the db. e.g. what happens when another query reads name and forgets this? we have the capability to just correct the data model so we probably should.

Reply:
I added a migration to do this.